### DISCUSSION TITLE
为什么我选择的打包工具是webpack，但是会提示我vite安装的有问题？

### DISCUSSION THREAD

**Discussion by cander0815** [vuepress/core#1284] [Q&A] [answered]
我使用的都是最新的bate61的包，运行时提示esbuild安装的有问题
配置入下图：
![image](https://user-images.githubusercontent.com/23468736/225223968-0c76ae55-75e8-4898-a4e3-bf4a3f1adb20.png)
报错信息入下图：
![image](https://user-images.githubusercontent.com/23468736/225224112-186dd5f9-1cec-452b-ad5f-2334e35c949b.png)

**Comment by Mister-Hope** [comment]
🫤报错信息的哪句话或者哪个词是和vite相关的？

**Reply by cander0815** [reply, marked as answer]
esbuild 应该是vite用到的打包工具，问题已经找到了 webpack版本应该 vuepress-webpack 来作为主包，我搞错了
